drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -177,6 +177,15 @@ def file(id:)
       Hmis::File.viewable_by(current_user).find_by(id: id)
     end
 
+    # don't paginate since we don't expect many records passed (todo @martha - discuss, maybe not ideal)
+    field :file_blobs, [Uploads::DirectUploadType], null: true do
+      argument :ids, [ID], required: true
+    end
+    def file_blobs(ids:)
+      # todo @martha PERMISSION LEAK
+      ActiveStorage::Blob.find(ids)
```

Comment:
@gigxz Here is a sketch of the approach we discussed last time for how we would display previously saved-but-not-submitted assessment files.

There are some bugs to unravel with this approach from the frontend, but before doing so, I'd like to discuss what I see as (potentially) a fatal flaw to the approach, which is: at the point when we have uploaded a DirectUpload blob, but have not yet saved a file record to the DB, it's difficult [not possible?] to track down what client/enrollment this file is associated with, and by extension, difficult [impossible?] to appropriately gate permissions for any endpoint where we directly expose such blobs.

I'm not sure, maybe there is a simple way around this that I'm just not understanding? But to me it is starting to look like we are required to save the file as a file record on save-without-submit, and then return it on subsequent fetches of the assessment. That approach is messy too, because it requires circumventing a bunch of form processor stuff that normally only runs on submit. But I can't think of a cleaner way to get the behavior we want without a permission leak.

Reply:
Decision from pairing: only support WIP saving for non file assessments. New API field `supports_save_in_progress`. Make it also return false if assessment has already been submitted

Reply:
@martha great point, thank you for catching that! I agree this would expose a potential leak. I also agree that adding a processing step to "save in progress" with a partial save (saving only Files) is a can of worms.

I think our best course of action right now may be to disallow saving files as "in progress" as discussed